derive/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "prost-serde-derive"
 version = "0.1.5"
-author = ["Park Joon-Kyu <segfault87@gmail.com>"]
+authors = ["Park Joon-Kyu <segfault87@gmail.com>"]
```

Comment:
How about adding your name here as well?

Reply:
:blobaww: Ok, I will add it with adding new tests.